apps/backend/src/authn/ldap.strategy.ts

```diff
@@ -37,12 +37,22 @@ export class LDAPStrategy extends PassportStrategy(Strategy, 'ldap') {
           user,
           configService.get('LDAP_MAILFIELD') || 'mail'
         );
-        req.user = this.authnService.validateOrCreateUser(
-          email,
-          firstName,
-          lastName,
-          'ldap'
-        );
+        // If user has multiple emails
+        if (typeof email === 'object') {
+          req.user = this.authnService.validateOrCreateUser(
+            email[0],
```

Comment:
Shouldn't this just be `typeof email === object ? email[0] : email` or something similar instead of duplicating the whole block?